prep.sh

```diff
@@ -59,3 +59,4 @@ git submodule update --init --remote
 copy_docs cumulus-etl etl 10
 copy_docs cumulus-library library 11
 copy_docs chart-review chart-review 12
+copy_docs cumulus-aggregator aggregator 13
```

Comment:
This is extremely subjective of course, but I'd vote to place aggregator between library and chart-review.

ETL -> Library -> Aggregator is a reasonable sequence. Chart Review is just kind of a hanging chad / bonus feature.